tutorials/actility-lorawan-operator/index.mdx

```diff
@@ -9,83 +9,83 @@ tags: Iot-Hub LoRaWAN© Loriot
-    - **Account prefix**: Enter `my-app`. This allows you to set the MQTT topic prefix. The messages will be published with under a topic such as the following: `<prefix>/things/<LoRaWAN device ID>/uplink`. 
-    - **Protocol**: `SSL`.
-    - **MQTT Username**: Your IoT Hub device ID.
-    - **MQTT Password**: Leave this field empty.
-    - **Certificate**: The device certificate file (ex: `my-device-crt.pem`).
-    - **Private Key**: The device key file (ex: `my-device-key.pem`).
-4. Click **Save**.
-5. Verify if the **Deployment Status** is set to **Opened** in the application page to make sure the application is running. 
+1. Navigate to the [Actility Thingpark Enterprise console](https://community.thingpark.io/).
+2. Select **Applications > Create** and opt for MQTT.
+3. Fill in the required parameters (unlisted settings can be customized):
+    - **Hostname**: Enter your IoT Hub endpoint, followed by the MQTT port (e.g., `iot.fr-par.scw.cloud:8883`).
+    - **Account prefix**: Specify `my-app` to set the MQTT topic prefix, leading to messages published under a topic like `<prefix>/things/<LoRaWAN device ID>/uplink`.
+    - **Protocol**: Choose `SSL`.
+    - **MQTT Username**: Input your IoT Hub device ID.
+    - **MQTT Password**: Leave this field blank.
+    - **Certificate**: Attach the device certificate file (e.g., `my-device-crt.pem`).
+    - **Private Key**: Provide the device key file (e.g., `my-device-key.pem`).
+4. Save your settings.
+5. Ensure the **Deployment Status** reflects **Opened** in the application page to confirm the application's operational status.
 
-    Your LoRaWAN© network is now connected to your IoT Hub.
+Your LoRaWAN© network is now seamlessly integrated with your IoT Hub.
 
-## Subscribe to the LoRaWAN© devices messages
+## Subscribing to LoRaWAN© device messages
 
-To ensure messages are being properly sent, we will create a new device that will subscribe to the LoRaWAN© devices messages.
+To ensure a smooth flow of messages, create a new device dedicated to subscribing to LoRaWAN© device messages.
 
-1. Create another IoT Hub device, but this time choose the `Allow Insecure` connections setting. Secured connections are not supported over Web Sockets, which the `MQTT Webclient` uses.
-2. Click **MQTT Webclient** on the device page. The client will open and connect automatically.
-    <Lightbox src="scaleway-mqtt_webclient.webp" alt="" />
-3. Subscribe to the LoRaWAN© devices messages:
-    - Click **Add New Topic Subscription** in the **Subscriptions** block
-    - Pick any color to identify this subscription
-    - Leave QoS 0
-    - Use `my-app/things/+/uplink` as the topic. This will instruct the IoT Hub to send you uplink messages from all devices
-    - Click **Subscribe**
+1. Create another IoT Hub device, this time enabling the **Allow Insecure** connections setting since secured connections are incompatible with Web Sockets, used by the **MQTT Webclient**.
```

Comment:
```suggestion
1. Create another IoT Hub device, now enabling the **Allow Insecure** connections setting since secured connections are incompatible with Web Sockets, used by the **MQTT Webclient**.
```